Symfony/CS/Fixer/PSR2/FunctionCallSpaceFixer.php

```diff
@@ -50,6 +51,15 @@ public function fix(\SplFileInfo $file, Tokens $tokens)
                 continue;
             }
 
+            // check for ternary operator
+            $endParenthesisIndex = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $index);
+            $nextNonWhiteSpace = $tokens->getNextMeaningfulToken($endParenthesisIndex);
+            if (!empty($nextNonWhiteSpace) && $tokens[$nextNonWhiteSpace]->equals('?')) {
+                if ($tokens[$lastTokenIndex]->isGivenKind($languageConstructionTokens)) {
```

Comment:
Try 1 && 2 && 3 at first but this condition become very long and it is hard to read. 
If you demand on second case i can change it.